src/ingest_validation_tools/docs_utils.py

```diff
@@ -170,7 +170,20 @@ def _make_constraints_table(field):
     if len(table_md_rows) < 3:
         # Empty it, if there is no data.
         table_md_rows = []
-    return '\n'.join(table_md_rows)
+    main_table_md = '\n'.join(table_md_rows)
+
+    onto_table_md = _make_onto_table(field['constraints']['enum']) \
+        if 'constraints' in field and 'enum' in field['constraints'] else ''
+    return main_table_md + onto_table_md
+
+
+def _make_onto_table(enum):
```

Comment:
maybe spell `ontology` - it almost reads like a typo into -> onto

Reply:
fixed